### PR TITLE
bug: Fix Rerouting

### PR DESCRIPTION
**Pull request by Tritium-VLK**
- Set total_incentives to 0 on source pool
- Set - reroute incentives on source pool

**Also changed to use a non-main branch for reroute config, change before merging**

### CODE REVIEW

**Comment by Tritium-VLK** [comment]
See the CSV in #124  for results
https://github.com/BalancerMaxis/protocol_fee_allocator/blob/512d3b69a8e80c6821cd4a00ca2f4147933e5e36/fee_allocator/allocations/incentives_2024-02-01_2024-02-15.csv

File: fee_allocator/accounting/settings.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ class Chains(Enum):
 CORE_POOLS_URL = "https://raw.githubusercontent.com/BalancerMaxis/multisig-ops/main/config/core_pools.json"
 
 REROUTE_CONFIG_URL = (
-    "https://raw.githubusercontent.com/BalancerMaxis/multisig-ops/main/config/core_pools_rerouting.json"
+    "https://raw.githubusercontent.com/BalancerMaxis/multisig-ops/rerouting/config/core_pools_rerouting.json"
```
Comment (Xeonus) [inline fee_allocator/accounting/settings.py:23]: Make sure to first merge multi-sig ops rerouting branch to main again but seems to work from that branch

File: fee_allocator/accounting/distribution.py [hunk 2]
```diff
@@ -186,15 +186,19 @@ def re_route_incentives(
...
+                raise Exception(f"Total Incentive from data {_data['total_incentives']} does not match aura + bal incentives {_total_incentives}")
             incentives[reroute[chain.value][pool_id]]['total_incentives'] += _total_incentives
             # Mark source pool incentives as rerouted
-            incentives[reroute[chain.value][pool_id]]['reroute_incentives'] += _data[
-                'total_incentives']
+            incentives[reroute[chain.value][pool_id]]['reroute_incentives'] += _total_incentives
+            # Zero out source pool
             incentives[pool_id]['aura_incentives'] = 0
             incentives[pool_id]['bal_incentives'] = 0
+            incentives[pool_id]['total_incentives'] = 0
```
Comment (Xeonus) [inline fee_allocator/accounting/distribution.py:202]: Agreed on this to properly reset all relevant columns

**Review by Xeonus — APPROVED** [review]
Logic makes sense and output comparisons resolve nicely: https://docs.google.com/spreadsheets/d/16axBQBVNdPssQyjcIyb_4m4vOb5rKISIlrVMk7A2qjs/edit#gid=1349161358

**Pull request merged**